### ISSUE TITLE
The model predicts pixels as opposed to true pixels

### ISSUE THREAD

**Issue by SorryMaker511** [jeya-maria-jose/KiU-Net-pytorch#37]
Hello,I have a strange question. When I use the trained model for segmentation, I can split the liver and the background, but the split liver has 0 pixels and the background is 1. In the original test set, the liver has 1 pixel and the background is 0.
